assets/l10n/app_ar.arb

```
@@ -329,7 +329,9 @@
   "enter_additional_info_req_id": "أدخل معرف طلب المعلومات الإضافية",
   "additional_info_req_id": "معرف طلب المعلومات الإضافية",
   "no_access_to_this_page": "ليست لديك صلاحية الوصول إلى هذه الصفحة",
+  "server_unreachable": "خادم المصادقة غير متاح. يرجى المحاولة مرة أخرى لاحقًا",
   "scheduled_job_settings": "إعدادات المهمة المجدولة",
+  "server_unreachable": "خادم المصادقة غير متاح. يرجى المحاولة مرة أخرى لاحقًا",
```

RuboCop (1.81.7)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)